contracts/Staking.sol

```diff
@@ -561,6 +561,9 @@ contract OlympusStaking is Ownable {
     address public warmupContract;
     uint public warmupPeriod;
 
+    event Stake(uint _amount, address _recipient);
```

Comment:
```suggestion
    event Stake(address indexed recipient, uint amount);
```

Reply:
I actually followed the original ohm setting
```
# callHandlers:
#   - function: stake(uint256,address)
#     handler: handleStake
#   - function: unstake(uint256,bool)
#     handler: handleUnstake
```

Reply:
And as far as I know, the eventHandler really for rinkeby testing purpose, we may not need those events for mainnet.